### DISCUSSION TITLE
initSession in custom component dev env

### DISCUSSION THREAD

**Discussion by dgalv** [writer/writer-framework#167] [Q&A]
Hello,

I'm in trouble, I don't understand how does the initSession() method fetch the "./api/init" URL ?
I get an error "ECONNREFUSED" instead of get a json content
How fix my problem ?

Thx

**Comment by FabienArcellier** [comment]
Hi,

Do you have this message on the terminal ?

```text
8:17:41 PM [vite] ws proxy error:
Error: connect ECONNREFUSED 127.0.0.1:5000
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1159:16)
```

If you have it, that mean you run only the dev environment for the frontend side. You have to run 2 terminal, one to run the backend with `streamsync edit apps/hello --port 5000` and one to run the frontend with `npm run custom.dev`
